libs/domain/cart/totals/components/discount/discount.component.spec.ts

```diff
@@ -108,8 +107,8 @@ describe('CartTotalsDiscountComponent', () => {
 
     it('should render label and subtotal', () => {
       const spans = element.shadowRoot?.querySelectorAll('span');
-      expect(spans?.[0].textContent).toBe('Discount');
-      expect(spans?.[1].textContent).toBe('price');
+      expect(spans?.[0].textContent).toContain('1 discounts');
```

Comment:
Should it be `.toContain('1 discount');` (singular)

Reply:
And what about adding a test for 2+ discounts?

Reply:
we do not have any i18n labels integrated into unit tests, hence there's no plurarlisation to be tested in unit tests